<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Chris York</author>
 <class>qspellingisDockBase</class>
 <widget class="QDockWidget" name="qspellingisDockBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QspellinGIS</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="fra_main">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTabWidget" name="Tabs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>461</width>
         <height>341</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Candara</family>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_layers">
        <attribute name="title">
         <string>Layer selection</string>
        </attribute>
        <widget class="QgsMapLayerComboBox" name="MapLayer">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>261</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QgsFieldComboBox" name="FieldOrderBy">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>50</y>
           <width>261</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QRadioButton" name="rad_action_pan">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>110</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Pan</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="rad_action_zoom">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>110</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelLayer">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>151</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Select visible layer</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="labelOrderBy">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>141</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Order by</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chk_use_sel">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>80</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Use selected only</string>
         </property>
        </widget>
        <widget class="QLabel" name="Canvas">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>110</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Canvas</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="setUp">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>451</width>
           <height>171</height>
          </rect>
         </property>
         <property name="title">
          <string>Q Spell In GIS</string>
         </property>
         <widget class="QComboBox" name="lang">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>60</y>
            <width>141</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="Language">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>111</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Language</string>
          </property>
         </widget>
         <widget class="QComboBox" name="setSpell">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>25</y>
            <width>141</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="Spelling_library">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>25</y>
            <width>111</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Spelling library</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pwl_select">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>281</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Personal Word List</string>
          </property>
         </widget>
         <widget class="QLabel" name="pwl_filename">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>281</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="byod_Filename">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>60</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Bring your own dictionary</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="feedback">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>20</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Bring your own dictionary</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="labelLogo">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>90</y>
            <width>120</width>
            <height>75</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Candara</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>QspellinGIS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QRadioButton" name="rad_action_identify">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>110</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Identify</string>
         </property>
        </widget>
        <widget class="QPushButton" name="layerRefresh">
         <property name="geometry">
          <rect>
           <x>415</x>
           <y>8</y>
           <width>33</width>
           <height>33</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Refresh layers</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_spelling">
        <attribute name="title">
         <string>Spell checking</string>
        </attribute>
        <widget class="QScrollArea" name="scrollArea">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>56</y>
           <width>451</width>
           <height>241</height>
          </rect>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>449</width>
            <height>239</height>
           </rect>
          </property>
          <widget class="QWidget" name="formLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>451</width>
             <height>231</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout"/>
          </widget>
         </widget>
        </widget>
        <widget class="QPushButton" name="btn_last">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>29</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>&gt;|</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_next">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>29</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_prev">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>29</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_first">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>29</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>|&lt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="CountFeatures">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>29</y>
           <width>131</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>- of -- features</string>
         </property>
        </widget>
        <widget class="QPushButton" name="save_record">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>0</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Accept</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cancel_record">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>30</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
        <widget class="QLabel" name="layer_label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>371</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Layer:</string>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
        <widget class="QPushButton" name="identify_feature">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>29</y>
           <width>61</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Identify</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
